generated/rflx-arrays-generic_inner_message.ads

```diff
@@ -216,14 +216,18 @@ is
 
    generic
       with procedure Process_Payload (Payload : out Types.Bytes);
+      with function Check_Length_Payload (Length : Types.Length) return Boolean;
```

Comment:
I think `Valid_Payload_Length` is better readable in the precondition and more consistent to existing functions (`Valid_Next`, `Valid_Predecessor`, ...).

Reply:
I tried to keep the naming scheme of having the field name last. Otherwise I would have named it `Check_Payload_Length`. Although you're right that `Valid_*` is more fitting. How about `Valid_Length_Payload`?

Reply:
What is the advantage of having the field name last? I think `Payload_Length` reads better. Actually, we could also consider removing the field name (for both generic subprograms).